index.js

```diff
@@ -38,9 +38,10 @@ export class Dagula {
    */
   async * get (cid, options = {}) {
     cid = typeof cid === 'string' ? CID.parse(cid) : cid
+    const order = options.order ?? 'rnd'
     log('getting DAG %s', cid)
     let cids = Array.isArray(cid) ? cid : [cid]
-    const search = options.search || breadthFirstSearch()
+    const search = options.search || blockLinks()
```

Comment:
Ah right, the half-formed idea here was that the `search` fn could be used to let a caller pass in a stateful depth first search equivalent to the default "just return all the links in the current block", but i didn't try and implement it to try out that idea.

...all i really needed at the time was a way to pass in a rule for filtering out some links so i could implement ~~car-scope: file~~ dag-scope: entity for a hamt and got carried away.

`search` could go away in favour of `linkFilter: ([name, cid]) => boolean`  or similar, and we could just have this get fn always do depth first. There was no strong reason for the previous default, and we can return depth-first for `rnd` as well.